libminifi/include/utils/Void_t.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
```

Comment:
I feel it a bit of overkill to have a separate file for such a tiny utility. I would just put `void_t` in GeneralUtils.h next to the other polyfills.

Reply:
Moved to `GeneralUtils.h`